src/Shipping_Method/Notice.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Non‑dismissible admin banner for PostNL
+ *
+ * @package PostNLWooCommerce
+ */
+
+namespace PostNLWooCommerce\Shipping_Method;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Notice
+ */
+class Notice {
```

Comment:
I think renaming it to Survey is better since this class is not for adding the notice only
```suggestion
class Survey {
```

and also adding `Leave review` button here will help
<img width="854" height="139" alt="image" src="https://github.com/user-attachments/assets/ece264d5-9c7b-40d4-a86c-7f703feb990b" />